core-plugins/docs/AzureBlobStore-batchsource.md

```diff
@@ -1,34 +1,38 @@
-# Azure Blob Store Batch Source
+# Microsoft Azure Blob Storage Batch Source
 
 Description
 -----------
 
-Batch source to use Azure Blob Store as a source.
+Batch source to use Microsoft Azure Blob Storage as a source.
 
 Use Case
 --------
 
-This source is used whenever you need to read from a Azure Blob Store.
-For example, you may want to read in files from Azure Blob Storage, parse them and then store
-them in a Microsoft SQL Server Database.
+This source is used whenever you need to read from Microsoft Azure Blob Storage. For
+example, you may want to read in files from Microsoft Azure Blob Storage, parse them and
+then store them in a Microsoft SQL Server Database.
 
 Properties
 ----------
 
-**path:** The path on Azure Blob Storage to use as input.
+**path:** The path on Microsoft Azure Blob Storage to use as input.
 
-**account:** The Microsoft Azure Storage account to use.
+**account:** The Microsoft Azure Blob Storage endpoint, with the Microsoft Azure Storage
```

Comment:
Ok, I just checked https://azure.microsoft.com/en-us/documentation/articles/hdinsight-hadoop-use-blob-storage/ as well, it seems to indicate that account name is just the first part like you call out. Since we expect the full name containing blob.core.windows.net, maybe lets just make sure that we convey that this is the full name.